.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,71 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  schedule:
+    - cron: '44 12 * * 3'
+
+jobs:
+  analyze:
+    name: Analyse
```

Comment:
Why did you change this to "Analyse" instead of the original "Analyze"?

Reply:
Because I'm a linguist, and etymologically "analyze" is completely incorrect spelling! The root is Greek -lysis, as in analysis, dialysis, and is not the same root as other -ise/ize verbs in English, French, and Romance languages. OED is very specific that "analyze" is wrong even in US English (though you see it everywhere)!

This is just me bing a pedant. 😯

Reply:
I had no idea and trust you 100% on this